kafka_consumer/CHANGELOG.md

```diff
@@ -2,6 +2,12 @@
 
 <!-- towncrier release notes start -->
 
+## 4.4.0 / 2024-06-11
+
+***Added***:
+
+* Update confluent_kafka and librdkafka ([#17726](https://github.com/DataDog/integrations-core/pull/17726))
```

Comment:
```suggestion
* Update `confluent_kafka` and `librdkafka` ([#17726](https://github.com/DataDog/integrations-core/pull/17726))
```

Reply:
Hello! I'll address this in another PR. Thanks for flagging.